### ISSUE TITLE
[Bug] catalog.json is generated empty

### ISSUE THREAD

**Issue by push2prod** [oracle/dbt-oracle#125]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

dbt generate docs succeeds but returns an empty catalog
{
 "nodes": {},
 "sources": {},
 "errors": null
}

### Expected Behavior

in previous version 1.6.0 with dbt-core 1.6.8 the catalog was generated correctly

### Environment

```markdown
- OS: Debian Gnu/Linux 11 (bullseye)
- Python: 3.9.16
- dbt: 1.7.4
```

### What Oracle database version are you using dbt with?

19c

**Comment by aosingh** [comment]
@push2prod thanks for reporting this. I was able to reproduce the problem and currently investigating the root cause. I will let you know once I have a fix for it.

**Comment by aosingh** [comment]
@push2prod Fix is released in `dbt-oracle==1.7.1rc1`. You can install the release candidate version using pip:

```
python3 -m pip install dbt-oracle==1.7.1rc1
```

Could you test and confirm if it works for you ? I can release `dbt-oracle==1.7.1` on PyPI after your confirmation

**Comment by push2prod** [comment]
Hey thanks for the fast response! Was able to check it only now, but the catalog is now generated fully, seems good for a merge
